packages/components/src/docs/General/11-FAQ.mdx

```diff
@@ -110,6 +153,30 @@ import 'https://solid-design-system.fe.union-investment.de/x.x.x/components/es/a
   </span>
 </sd-accordion>
 
+<sd-accordion>
+  <h3 className="text-base sb-unstyled" slot="summary">
+    Can I change the appearance of the components of the Solid Design system?
+  </h3>
+  <span className="sb-unstyled">
+    Yes, the Solid Design System supports skinning. You can customise the appearance of components through styles to
```

Comment:
It should be clear, that skinning is especially focused on changing colors.

In addition there are part selectors, that are meant to optimize for specific needs.

Maybe these should be two things: One is regarding "theming", the other one regarding "customization".